elyra/cli/pipeline_app.py

```diff
@@ -177,8 +192,7 @@ def _validate_pipeline_definition(pipeline_definition):
     issues = validation_response.to_json().get('issues')
     _print_issues(issues)
 
-    if validation_response.has_fatal:
-        raise click.ClickException("Pipeline validation FAILED. The pipeline was not submitted for execution.")
+    return validation_response.has_fatal
```

Comment:
I'll flip the return value, but I believe we should try to come up with a new method name that better describes what it does. Something like `is_valid...`  wouldn't really work because the method's implementation has a side effect (printing issues). Any ideas? 

(Issues like these are why I added the todo to clean up the existing code to make it flow better, but decided against it for now due to time constraints.)